plugins/drive-resources/src/index.ts

```diff
@@ -50,8 +50,13 @@ async function EditDrive (drive: Drive): Promise<void> {
 async function DownloadFile (doc: File | File[]): Promise<void> {
   const files = Array.isArray(doc) ? doc : [doc]
   for (const file of files) {
-    const url = getFileUrl(file.file, 'full', file.name)
-    window.open(url, '_blank')
+    const href = getFileUrl(file.file, 'full', file.name)
+    const link = document.createElement('a')
```

Comment:
You should remove it after

Reply:
nope, it is not required because I did not add the element into the DOM